cumulus/queue/slurm.py

```diff
@@ -64,21 +64,26 @@ def job_statuses(self, jobs):
         output = self._cluster_connection.execute('squeue -j %s'
                                                   % job_ids)
 
+        return self._extract_job_statuses(output, jobs)
+
+    def to_job_queue_state(self, slurm_state):
+        state = None
+        if slurm_state in SlurmQueueAdapter.RUNNING_STATE:
+            state = JobQueueState.RUNNING
+        elif slurm_state in SlurmQueueAdapter.ERROR_STATE:
+            state = JobQueueState.ERROR
+        elif slurm_state in SlurmQueueAdapter.QUEUED_STATE:
+            state = JobQueueState.QUEUED
+        elif slurm_state in SlurmQueueAdapter.COMPLETE_STATE:
+            state = JobQueueState.COMPLETE
+
```

Comment:
Not sure, I understand what you mean,  SlurmQueueAdapter.COMPLETE_STATE for example is a list rather than a single value?